dev/workflow/rocoto/gefs_tasks.py

```diff
@@ -10,6 +10,22 @@ def __init__(self, app_config: AppConfig, run: str) -> None:
 
     def stage_ic(self):
 
+        stage_ic_map = {'gefs-offline': self._gen_stage_ic,
```

Comment:
Minor name change suggestion to match `gefs-offline`
```suggestion
        stage_ic_map = {'gefs-offline': self._offline_stage_ic,
```